pkg/forwarder/forwarder.go

```diff
@@ -376,6 +378,13 @@ func NewDefaultForwarder(options *Options) *DefaultForwarder {
 	return f
 }
 
+func getAgentFolder(options *Options) string {
+	if HasFeature(options.EnabledFeatures, CoreFeatures) {
+		return "core"
+	}
+	return ""
+}
```

Comment:
I'm guessing we would add other agent to the list later/when needed ?

Reply:
Yes, @truthbk  added `HasFeature` to enable features based on the Agent (core, process ...).